deploy.js

```diff
@@ -0,0 +1,150 @@
+const { SkynetClient: SkynetClientNode } = require("@skynetlabs/skynet-nodejs");
+const { SkynetClient, genKeyPairFromSeed } = require("skynet-js");
+
+const { cyan } = require("chalk");
```

Comment:
if this is going to be our official copy & paste deploy script we should minimise the number of dependencies we add, namely we can safely remove chalk here